Library/Homebrew/cmd/test-bot.rb

```diff
@@ -807,64 +800,62 @@ def test_ci_upload(tap)
     safe_system "git", "reset", "--hard", "origin/master"
     safe_system "brew", "update"
 
-    if pr
-      if ARGV.include?("--legacy")
-        pull_pr = "https://github.com/Homebrew/homebrew/pull/#{pr}"
-        safe_system "brew", "pull", "--clean", "--legacy", pull_pr
-      else
-        pull_pr = "https://github.com/#{tap.user}/homebrew-#{tap.repo}/pull/#{pr}"
-        safe_system "brew", "pull", "--clean", pull_pr
-      end
-    end
```

Comment:
Why this can be removed now? For example, how can merge work if you don't pull the new formula first? Sorry, if i'm missing something.